policyengine_us/variables/gov/states/ca/tax/income/deductions/itemized/ca_itemized_deductions.py

```diff
@@ -12,3 +12,17 @@ class ca_itemized_deductions(Variable):
         "https://www.ftb.ca.gov/forms/2022/2022-540.pdf"
     )
     defined_for = StateCode.CA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        itm_deds = [
+            deduction
+            for deduction in p.itemized_deductions
+            if deduction not in ["salt_deduction"]
+        ]
+        itm_deds_less_salt = add(tax_unit, period, itm_deds)
+        property_taxes = add(tax_unit, period, ["real_estate_taxes"])
+        salt = p.itemized.salt_and_real_estate
+        cap = salt.cap[tax_unit("filing_status", period)]
```

Comment:
I don't think California caps SALT-less-state-income-tax.

From https://www.ftb.ca.gov/forms/2022/2022-540-ca.pdf
<img width="735" alt="image" src="https://user-images.githubusercontent.com/6076111/225515804-7e7804c9-143b-4881-946e-b0bf6f832173.png">

and https://www.ftb.ca.gov/forms/2022/2022-540-ca-instructions.html

>Line 5e – The federal deduction for state and local tax is limited to $10,000 ($5,000 for married filing separate) for the aggregate of state and local income taxes and property taxes. California does not conform. If your deduction was limited under federal law, enter an adjustment on line 5e, column C for the amount over the federal limit.

Reply:
Done